js/views/messagesitem.js

```diff
@@ -98,6 +98,8 @@ define(function(require) {
 		openMessage: function(event) {
 			event.stopPropagation();
 			$('#mail-message').removeClass('hidden-mobile');
+			// make sure message is marked as read when clicked on it
+			Radio.ui.trigger('messagesview:messageflag:set', this.model.id, 'unseen', false);
```

Comment:
We can do it either way, I'd just have to change this one line again in the other PR if this one gets merged first.